gcp-common/src/main/java/io/micronaut/gcp/credentials/GoogleCredentialsFactory.java

```diff
@@ -83,11 +83,11 @@ public GoogleCredentialsFactory(@NonNull GoogleCredentialsConfiguration configur
      * @return The {@link GoogleCredentials}
      * @throws IOException An exception if an error occurs
      */
-    @Requires(missingBeans = GoogleCredentials.class)
     @Requires(classes = com.google.auth.oauth2.GoogleCredentials.class)
```

Comment:
```suggestion
    @Requires(classes = GoogleCredentials.class)
```

I think this is all the same class. Having the fully qualified name makes me think there's different ones, but I don't believe there is 🤔

Reply:
You are correct, they are all the same.